exercises/bowling/example.go

```diff
@@ -29,63 +37,62 @@ func NewGame() *Game {
 // An error is possible depending on pin value and previous rolls.
 func (g *Game) Roll(pins int) error {
 	// Validate pin count on roll.
-	if pins > 10 {
+	if pins > pinsPerFrame {
 		return ErrPinCountExceedsPinsOnTheLane
 	}
 	if pins < 0 {
 		return ErrNegativeRollIsInvalid
 	}
-	if g.completedFrames() == 10 {
-		// Already completed 10 frames.
+	if g.completedFrames() == framesPerGame {
 		return ErrCannotRollAfterGameOver
 	}
 	// Record the roll.
 	g.rolls[g.nRolls] = pins
 	g.nRolls++
-	if pins == 10 && g.completedFrames() < 9 {
-		// Frames 1 .. 9 can be strikes with no problems.
+	if pins == pinsPerFrame && g.completedFrames() < (framesPerGame-1) {
```

Comment:
The parenthesis tend to highlight the expression.  But it is probably more idiomatic to leave them out.